testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -941,9 +941,9 @@ def ref_noPBC(self):
                                 dtype=np.float32),
             'BSph': (173.40482, np.array([4.23789883, 0.62429816, 2.43123484], dtype=np.float32)),
             'PAxes': np.array([
-                [-0.78787867, -0.26771575, 0.55459488],
-                [0.40611024, 0.45112859, 0.7947059],
-                [0.46294889, -0.85135849, 0.24671249]])
+                [ 0.78787867,  0.26771575, -0.55459488],
```

Comment:
Updated the test PAxes to an array that returns 1 when checking the triple product via:
 `np.dot(np.cross(PAxes[0], PAxes[1]), PAxes[2])`

The original returned -1 (as I assume handedness wasn't checked before?).